src/main/js/ephox/imagetools/transformations/ImageTools.js

```diff
@@ -80,13 +82,13 @@ define("ephox/imagetools/transformations/ImageTools", [
 
   function resize(blob, w, h) {
     return Conversions.blobToImage(blob).then(function(image) {
-      var canvas = Canvas.create(w, h),
-        context = Canvas.get2dContext(canvas);
-
-      context.drawImage(image, 0, 0, w, h);
-      revokeImageUrl(image);
-
-      return Conversions.canvasToBlob(canvas, blob.type);
+      return ImageResizerWebgl.scale(image, w, h)
```

Comment:
Could we wrap them as: 
```
ImageTransformations.resize(blob, 100, 100, [
  ImageResizerWebgl.simple,
  ImageResizerCanvas.simple.
  ImageResizerWebgl.gradient,
  ImageResizerCanvas.gradient.
  ImageResizerWebgl.hermite,
  ImageResizerCanvas.hermite
]);
```

Or maybe that is overkill and to implementation specific.